tools/autotools/LICENSE

```diff
@@ -0,0 +1 @@
+/home/mathieut/Projects/codes/cp2k/LICENSE
```

Comment:
Better use a relative symlink.

Reply:
> I can also extend the formatting test to ensure that the generated configure script remains up to date.

Yes I think it is a good idea. you will need autoconf-archive installed. Any version will do. I did not want to version the configure script because it is easy to generate from the rest.